src/routes/movies.py

```
@@ -199,17 +199,17 @@ async def create_movie(
                 await db.flush()
             actors.append(actor)
 
-        languages = []
-        for language_name in movie_data.languages:
-            lang_stmt = select(LanguageModel).where(LanguageModel.name == language_name)
-            lang_result = await db.execute(lang_stmt)
-            language = lang_result.scalars().first()
+        director = []
+        for director_name in movie_data.directors:
+            director_stmt = select(DirectorModel).where(DirectorModel.name == director_name)
```

Ruff (0.11.9)

Local variable `director_stmt` is assigned to but never used

Remove assignment to unused variable `director_stmt`

(F841)